docs/source/usage/deployment.rst

```diff
@@ -0,0 +1,140 @@
+==========
+Deployment
+==========
+
+This guide uses Zoom as our service, your service may be different, be sure to name it something appropriately. 
+
+***************
+Getting Started
+***************
+
+  1. Clone the master branch of the repository at “https://github.com/mozilla/mozdef-event-framework” to the local file system.
+  2. Before any action, decide on the various parameter and environment variable values, as these are referenced throughout the CF templates and serverless framework configuration.
+     The following are the environment variables with example values:
+
+     * PROJECT: MozDef-EF
+     * SERVICE: zoom
+     * STAGE: dev
+     * API_PATH: events
+     * Stack name: zoom2mozdef
+
+************************
+Steps in the AWS Console
+************************
+
+You'll need to log into the AWS console or you can alternatively use the aws-cli to create the parameters needed for your webhook to properly authenticate:
+
+  1. Login to AWS Console using your account credentials. This framework has been tested to work with federated logins where the user assumes a role.
+  2. Navigate to “Services” and under “Management and Governance” select “Systems Manager”
+  3. Once the page loads, under “Application Management, choose “Parameter Store"
+  4. Create a parameter there in the form of:
+
+      “/<project>/<service>/<environment>/auth_token”
+
+      **Example:** For our Zoom example, this would be something like:
+       
+    .. image:: ../images/Parameter_Store_Example.png
+       :width: 400
+       :alt: /MozDef-EF/zoom/prod/auth_token
+
+  5. In the description field, add “Authentication token used by the Webhook app to post events”.
+  6. For type, select “SecureString”. 
+  7. For value, paste the value from the app’s configuration from your webhook's configured authentication token.
+  8. Add a Tag with key: Project and value: MozDef-EF-<environment>, then create the parameter.
+
+*************************************
+Fill in the Framework Config Template
+*************************************
+
+The following should be done in your local copy of the framework you cloned or forked:
+
+
+  1. Open the downloaded repository in an IDE to edit locally.
+  2. Under config directory, edit “buildspec-prod.yml” file to contain:
```

Comment:
If we are using `dev` as stage in the beginning of the file, we should change all stage/environment variables to "dev"

Reply:
Done